client/src/visualiser-src/sorting-visualiser/util/codeSnippets.ts

```diff
@@ -12,6 +12,37 @@ export const bubbleCodeSnippet = `void bubble_sort(int arr[], int arr_size) {
     }
 }`;
 
+export const mergeCodeSnippet = `void merge_sort(int arr[], int low, int high, int tmp[]) {
```

Comment:
Not sure if 'tmp' is needed? Could you just create the array in this function?
Have a look at the code [here](http://www.cse.unsw.edu.au/~cs2521/20T2/lecs/mergesort/slides.html#s1).

Reply:
On page 5/13 line4 in the code block, he also created a temp list. The difference between us is that he did it inside the merge sort implementation but I passed the temp list as a parameter. I think the basic idea is the same, it's just a matter of how it's implemented.